src/client/components/builder/questions/DropdownField.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import { BiListUl } from 'react-icons/bi'
+import {
+  Box,
+  HStack,
+  VStack,
+  Text,
+  Input,
+  Select,
+  Textarea,
+} from '@chakra-ui/react'
+
+import { useCheckerContext } from '../../../contexts'
+import { createBuilderField, QuestionFieldComponent } from '../BuilderField'
+import { BuilderActionEnum, ConfigArrayEnum } from '../../../../util/enums'
+
+const InputComponent: QuestionFieldComponent = ({ field, index }) => {
+  const { title, description } = field
+  const { dispatch } = useCheckerContext()
+
+  const updateTitleOrDescription = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target
+    dispatch({
+      type: BuilderActionEnum.Update,
+      payload: {
+        currIndex: index,
+        element: { ...field, [name]: value },
+        configArrName: ConfigArrayEnum.Fields,
+      },
+    })
+  }
+
+  const changeOptions = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    const text = e.target.value
+    const options = text.split('\n').map((label, value) => ({ label, value }))
+    field.options = options
+    dispatch({
+      type: BuilderActionEnum.Update,
+      payload: {
+        currIndex: index,
+        element: { ...field, options },
+        configArrName: ConfigArrayEnum.Fields,
+      },
+    })
+  }
+
+  return (
+    <HStack w="100%" alignItems="flex-start">
+      <Box fontSize="20px" pt={2}>
+        <BiListUl />
+      </Box>
+      <VStack align="stretch" w="90%" spacing={6}>
+        <VStack align="stretch" spacing={2}>
+          <Input
+            type="text"
+            name="title"
+            placeholder="Question"
+            onChange={updateTitleOrDescription}
+            value={title}
+          />
+          <Input
+            type="text"
+            name="description"
+            placeholder="Description"
+            onChange={updateTitleOrDescription}
+            value={description}
+          />
+        </VStack>
+        <VStack spacing={4} alignItems="left" w="50%">
+          <Textarea
+            value={field.options.map((o) => o.label).join('\n')}
+            onChange={changeOptions}
+            placeholder="Enter each option on a new line"
+          />
+        </VStack>
+      </VStack>
+    </HStack>
+  )
+}
+
+const PreviewComponent: QuestionFieldComponent = ({ field }) => {
+  const { title, description, options } = field
+  return (
+    <VStack align="stretch" w="100%" spacing={6}>
+      <VStack align="stretch">
+        <HStack>
+          <BiListUl fontSize="20px" />
+          <Text>{title}</Text>
+        </HStack>
+        {description && <Text color="#718096">{description}</Text>}
+      </VStack>
+      <Select>
+        {options.map(({ value, label }, i) => (
```

Comment:
Just wondering, do you think we should enforce some sort of sort order on the select options? A really long list (e.g. countries) would benefit from it. 

Along the same vein, should we try to create a select component for this with search, similar to Forms? This can be done with `downshift` and `match-sorter` like in [ExpressionInput](https://github.com/opengovsg/checkfirst/blob/develop/src/client/components/builder/logic/ExpressionInput.tsx).

Reply:
My worries are as follows:
- there might be a reason why the user may wish to impose his/her own sort order. even for something like a list of countries, the user may decide to prioritise countries where we expect a lot of traffic (see [The Oatmeal](https://theoatmeal.com/comics/shopping_cart))
- I think we could! My suspicion is that Chakra deliberately used the native form options in the interest of accessibility. If there is a way to do it such that we have a more consistent visual element while still fulfilling accessibility goals, we could invest some time in it